.travis.yml

```diff
@@ -38,7 +39,9 @@ jobs:
     - stage: check
       script: bundle exec fastlane assemble
     - stage: deploy
-      script: ./travis/deploy.sh
+      script:
+        - ./git_crypt_unlock.sh || travis_terminate 1
```

Comment:
I'm not sure whether deploy below will run in the same stage as this one.  If it doesn't then it's probably running in separate VMs.  I put it both places because i think travis might deploy here and the one below in separate stages.

Reply:
git_crypt_unlock.sh is already in `install:` so it will get done on every stage